.github/workflows/dart-tests.yaml

```diff
@@ -0,0 +1,90 @@
+name: Relic CI
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+      - add-dart-project-and-ci-setup # temporary, testing
```

Comment:
Question: Wouldn't a pull request towards main trigger this pipe?

Reply:
According to the rules in the next section?

Reply:
this is removed, it was just to test, i removed here: https://github.com/serverpod/relic/pull/1/commits/3b502758fb8bf3c045986625800f373130e53b91